src/services/identity/providers/studio/postgres.ts

```diff
@@ -660,9 +660,6 @@ export class PostgresIdentityService extends DefaultIdentityService {
 		customer: CustomerEntity
 	): Promise<CreateStatusList2021Result> {
 		const agent = await this.createAgent(customer);
-		if (!(await IdentifierService.instance.find({ did: did, customer: customer }))) {
-			throw new Error(`${did} not found in wallet`);
```

Comment:
I think these checks are required as we do not want to create a StatusList with did owned by someone else.

Reply:
They are moved a layer above to credential status service inorder to avoid duplicate calls. But there is a chance of error if we call it directly. I am looking into it